src/Symfony/Component/Console/Tests/Fixtures/application_renderexception_linebreaks.txt

```diff
@@ -1,5 +1,5 @@
 
-In ApplicationTest.php line 770:
+In ApplicationTest.php line 795:
```

Comment:
Going to be a nightmare, we should make the related tests more flexible by using a placeholder where they have been introduced

Reply:
Addressed in #25794